### ISSUE TITLE
MySQL integration

### ISSUE THREAD

**Issue by Periscope123** [tiangolo/full-stack-fastapi-postgresql#412]
Hello!

I have a question about MySQL integration. Right now we can generate a starter pack with PSQL config. How can I change it so my app will use MySQL? 

`PGAdmin for PostgreSQL database, you can modify it to use PHPMyAdmin and MySQL easily.` 

Is there any example project with MySQL integration?

**Comment by mythral0** [comment]
Yes, I also need to know how to do this and also in stead of just mysql also with mssql
I think this project would benaffit significantly with a little documentation explaining how to wire up other native (not docker) databases.
I have tried replacing
“engine = create_engine(settings.SQLALCHEMY_DATABASE_URI, pool_pre_ping=True)”
With 
“engine = create_engine("mysql+mysqlconnector://root:Tahera621@10.8.0.1:3306/motest")”

But this doesn’t allow the stack to start.

Help would be much appreciated.

**Comment by mythral0** [comment]
Yes, I also need to know how to do this and also in stead of just mysql also with mssql
I think this project would benaffit significantly with a little documentation explaining how to wire up other native (not docker) databases.
I have tried replacing
“engine = create_engine(settings.SQLALCHEMY_DATABASE_URI, pool_pre_ping=True)”
With 
“engine = create_engine("mysql+mysqlconnector://root:Tahera621@10.8.0.1:3306/motest")”

But this doesn’t allow the stack to start.

Help would be much appreciated.

**Comment by phamngocquy** [comment]
I'm using pymysql and it work well
“engine = create_engine("mysql+pymysql://root:Tahera621@10.8.0.1:3306/motest")”

**Comment by vuongtlt13** [comment]
May be this [repo](https://github.com/vuongtlt13/full-stack-fastapi-mysql) could help you

**Comment by nzivo** [comment]
@vuongtlt13 @tiangolo  I tried installing google-cloud-storage and I couldn't install it using poetry but pip does it in the system but it's not accessible by the docker project

**Comment by nzivo** [comment]
> @vuongtlt13 @tiangolo I tried installing google-cloud-storage and I couldn't install it using poetry but pip does it in the system but it's not accessible by the docker project

I ran it successfully after adding RUN bash -c "pip install google-cloud-storage" in `backend.dockerfile` and `celeryworker.dockerfile` immediately after the project.toml `COPY `command
